### PR TITLE
py-colorcet: add new package

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-colorcet/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    """A collection of perceptually acccurate 256-color colormaps for use with
+    Python plotting programs like Bokeh, Matplotlib, HoloViews, and Datashader."""
+
+    homepage = "https://colorcet.holoviz.org/index.html"
+    pypi     = "colorcet/colorcet-3.0.0.tar.gz"
+
+    maintainers = ['vvolkl']
+
+    version('3.0.0', sha256='21c522346a7aa81a603729f2996c22ac3f7822f4c8c303c59761e27d2dfcf3db')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-colorcet/package.py:20]: Can you also add a dep on Python 2.7+

File: var/spack/repos/builtin/packages/py-colorcet/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    depends_on('py-setuptools@30.3.0:', type='build')
+    depends_on('py-param@1.7.0:', type='build')
+    depends_on('py-pyct@0.4.4:', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-colorcet/package.py]: These are also required at run-time

File: var/spack/repos/builtin/packages/py-pyct/package.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+class PyPyct(PythonPackage):
+    """Python package common tasks for users (e.g. copy examples, fetch data, ...)"""
+
+    pypi     = "pyct/pyct-0.4.8.tar.gz"
+
+    maintainers = ['vvolkl']
+
+    version('0.4.8', sha256='23d7525b5a1567535c093aea4b9c33809415aa5f018dd77f6eb738b1226df6f7')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pyct/package.py:18]: Python 2.7+

File: var/spack/repos/builtin/packages/py-pyct/package.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    depends_on('py-setuptools@30.3.0:', type='build')
+    depends_on('py-param@1.7.0:', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pyct/package.py]: run-time

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-colorcet/package.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    Python plotting programs like Bokeh, Matplotlib, HoloViews, and Datashader."""
+
+    homepage = "https://colorcet.holoviz.org/index.html"
+    pypi     = "colorcet/colorcet-3.0.0.tar.gz"
+
+    maintainers = ['vvolkl']
+
+    version('3.0.0', sha256='21c522346a7aa81a603729f2996c22ac3f7822f4c8c303c59761e27d2dfcf3db')
+
+    depends_on('python@2.7:')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-colorcet/package.py]: Build/run

File: var/spack/repos/builtin/packages/py-pyct/package.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+class PyPyct(PythonPackage):
+    """Python package common tasks for users (e.g. copy examples, fetch data, ...)"""
+
+    pypi     = "pyct/pyct-0.4.8.tar.gz"
+
+    maintainers = ['vvolkl']
+
+    version('0.4.8', sha256='23d7525b5a1567535c093aea4b9c33809415aa5f018dd77f6eb738b1226df6f7')
+
+    depends_on('python@2.7:')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pyct/package.py]: Build/run

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — DISMISSED** [review]

**Comment by vvolkl** [comment]
Thanks for the comments!

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**